src/remoteConnector.ts

```diff
@@ -765,57 +349,15 @@ export class RemoteConnector extends Disposable {
 									if (action === seeLogs) {
 										this.logService.show();
 									} else if (action === showTroubleshooting) {
-										vscode.env.openExternal(vscode.Uri.parse('https://www.gitpod.io/docs/ides-and-editors/vscode#connecting-to-vs-code-desktop-ssh'));
+										vscode.env.openExternal(vscode.Uri.parse('https://www.gitpod.io/docs/references/ides-and-editors/vscode#connecting-to-vs-code-desktop'));
```

Comment:
@jeanp413 we should reference `https://www.gitpod.io/docs/references/ides-and-editors/vscode#troubleshooting` here, SSH Gateway is till relevant we cannot omit it here

Reply:
also left a note here: https://github.com/gitpod-io/website/pull/4001#pullrequestreview-1661373331

Reply:
```suggestion
										vscode.env.openExternal(vscode.Uri.parse('https://www.gitpod.io/docs/references/ides-and-editors/vscode#troubleshooting'));
```